WeakAuras/GenericTrigger.lua

```diff
@@ -1551,6 +1551,14 @@ function GenericTrigger.Add(data, region)
       end
     end
   end
+
+  if warnAboutCLEUEvents then
+    Private.AuraWarnings.UpdateWarning(data.uid, "spamy_event_warning", "warning",
+                L["COMBAT_LOG_EVENT_UNFILTERED with no filter can trigger frame drops in raid environment. Find more information:\nhttps://github.com/WeakAuras/WeakAuras2/wiki/Custom-Triggers#events"],
+                  true)
+  else
+    Private.AuraWarnings.UpdateWarning(data.uid, "spamy_event_warning")
```

Comment:
For every aura warning there should be in general something that clears the warning.

Also I'm not sure I agree on the choice to print the warning on the console. I forsee lots of feedback on that, and not all positive :)

Reply:
I think the benefit for users out-weight the annoyance, but that works only if they don't chose to ignore the message in chat, or don't know what to make of it